Content.Shared/Gravity/SharedGravityGeneratorComponent.cs

```diff
@@ -3,42 +3,45 @@
 
 namespace Content.Shared.Gravity;
 
-[NetworkedComponent()]
-[Virtual]
-public partial class SharedGravityGeneratorComponent : Component
+[RegisterComponent, NetworkedComponent, AutoGenerateComponentState(fieldDeltas: true)]
+public sealed partial class SharedGravityGeneratorComponent : Component
```

Comment:
```suggestion
public sealed partial class GravityGeneratorComponent : Component
```
rename this, same for the file name